examples/mobile/UIComponents/components/view-controls/main-tab/one-appbar.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../../lib/tau/mobile/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page">
+		<link href="./main-tab-sample.css" rel="stylesheet" />
+		<header>
+			<div class="ui-appbar-title-container">
+				<span class="ui-appbar-title">One</span>
+			</div>
+		</header>
+		<div class="ui-content">
+			<div class="ui-content-area app-full-height">
```

Comment:
How about adding the app-full-height to the "core.less"?
I think it is a necessary style.
I think 'fill-height' is better than 'app-full-height'.

Reply:
I'm not sure if the adding this to framework is good idea. We can do this in separate change.

About name convention... I want to distinct framework classes and classes specific for app.

Reply:
However, it is inefficient to add additional css in all examples for this small one. 
I think you can add the style you need. It is my personal option. Anyway, I got it.